largest-series-product/largest_series_product_tests.erl

```diff
@@ -6,3 +6,15 @@ three_test() -> ?assert( 504 =:= largest_series_product:from_string("0123456789"
 five_test() -> ?assert( 15120 =:= largest_series_product:from_string("0123456789", 5) ).
 
 six_test() -> ?assert( 23520 =:= largest_series_product:from_string("73167176531330624919225119674426574742355349194934", 6) ).
+
+all_zeroes_test() -> ?assert( 0 =:= largest_series_product:from_string("0000", 2) ).
+
+all_contain_zeroes_test() -> ?assert( 0 =:= largest_series_product:from_string("99099", 3) ).
+
+too_long_test() -> ?assertException(error, function_clause, largest_series_product:from_string("123", 4) ).
```

Comment:
I settled on using `assertError` with `_` as the `TermPattern` after reading http://erlang.org/doc/apps/eunit/chapter.html#EUnit_macros - basically I don't care what kind of error it is, I just want it to be an error. I think that's the way to go, but let me know if you disagree of course.